src/evaluations/evaluator.py

```diff
@@ -205,7 +205,12 @@ def __init__(self, evaluation_config, sketch_estimator_config_list, run_name,
           'SketchEstimatorConfig objects')
     assert isinstance(parallel_cores, int), (
         'parallel_cores must be an int')
-    self.parallel_cores = parallel_cores if parallel_cores >= 1 else cpu_count()
+    if parallel_cores > cpu_count():
+      self.parallel_cores = cpu_count()
+    elif parallel_cores >= 1:
+      self.parallel_cores = parallel_cores
+    else:
+      self.parallel_cores = cpu_count()
```

Comment:
probably it's clearer this way?
```suggestion
    self.parallel_cores = min(1, parallel_cores, cpu_count())
```